report/metrics_handles.go

```diff
@@ -38,15 +38,15 @@ var (
 func SetupWindowsHandlesMetrics(logger *logp.Logger, reg *monitoring.Registry) {
 	beatProcessSysInfo, err := sysinfo.Self()
 	if err != nil {
-		logger.Error("Error while getting own process info: %v", err)
+		logger.Errorf("Error while getting own process info: %v", err)
```

Comment:
```suggestion
		logger.Errorf("Error while getting own process info: %s", err)
```

Let's use %s for errors so we rely on fmt.Stringer.